lava/lava-job-definitions/shared/templates/bootbomb-recovery.yaml

```diff
@@ -47,20 +47,14 @@
            - cd /lava-lxc
            - dpkg -i imx-usb-loader_0-git20181105.4aa98090-1_amd64.deb
            - lsusb
+           - grep 15a2 /sys/bus/usb/devices/*/idVendor
            - DEVICE_PATH=$(grep 15a2 /sys/bus/usb/devices/*/idVendor |grep "$LAVA_STORAGE_INFO_0_SATA" |awk '{split($0,a,"/idVendor"); print a[1]}')
            - BUSNUM=$(cat $DEVICE_PATH/busnum)
            - DEVICENUM=$(cat $DEVICE_PATH/devnum)
            - imx_usb --bus=$BUSNUM --device=$DEVICENUM {{ bootbomb_filename }}
-           - BLOCK_DEVICE=$(readlink /sys/class/block/sd* |grep $LAVA_STORAGE_INFO_0_SATA |awk '/[b-z]$/{print $1}' |awk '{n=split($0,a,"/"); print a[n]}')
-           - while [ -z "$BLOCK_DEVICE" ]
-           - do
-           -    sleep 10
-           -    BLOCK_DEVICE=$(readlink /sys/class/block/sd* |grep $LAVA_STORAGE_INFO_0_SATA |awk '/[b-z]$/{print $1}' |awk '{n=split($0,a,"/"); print a[n]}')
-           - done
-           - lsblk
-           - fdisk -l
-           - cd /lava-lxc
-           - bmaptool --quiet copy --bmap /lava-lxc/*.wic.bmap /lava-lxc/*.wic.gz /dev/$BLOCK_DEVICE
+           - sleep 30
```

Comment:
Can you add a comment on the reason of the sleep?

Reply:
comment added.